src/BasePaintThemeRegistry.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract BasePaintThemeRegistry is Ownable {
+    struct ThemeData {
+        string theme;
+        string[] palette;
```

Comment:
for palette, it makes sense to make this an array of `uint24`, i.e. 3 bytes chunks